srcs/runners_manager/vm_creation/VmType.py

```diff
@@ -1,18 +1,26 @@
+
+from marshmallow import Schema, fields
+
+
+class DefaultVmConfig(Schema):
+    image = fields.Str(required=True)
+    flavor = fields.Str(required=True)
+
+
```

Comment:
I don't like this. This is a openstack configuration template. 
We should take a dictionary in our object and have this template in the `openstack/schema.py` file.

Reply:
Then we import this class like the `OpenstackManager` and we create a Interface like `CloudManager`, or we condiser just checking the status and then use the directly the dictionary

Reply:
Been trying to do this today, but haven't got the time to finish yet. In progress